src/contracts/deposit/deposit.service.ts

```diff
@@ -196,7 +184,8 @@ export class DepositService {
   public async setCachedEvents(
     cachedEvents: VerifiedDepositEventsCache,
   ): Promise<void> {
-    return await this.cacheService.setCache({
+    await this.levelDBCacheService.deleteCache();
+    await this.levelDBCacheService.insertEventsCacheBatch({
       ...cachedEvents,
       headers: {
```

Comment:
is setCachedEvents used only in tests?
we still have version here, do we use it somewhere?

Reply:
fixed